docs/index.md

```diff
@@ -1,11 +1,58 @@
-# Github-runner operator
-A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators) deploying self-hosted GitHub runners.
-  
-Each unit of this charm will start a configurable number of LXD based containers and virtual machines to host them. Each runner performs only one
-job, after which it unregisters from GitHub to ensure that each job runs in a clean environment. The charm will periodically check the number of idle runners and spawn or destroy them as
-necessary to maintain the configured number of runners. Both the reconciliation interval and the number of runners to maintain are configurable.
+A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators) for deploying and managing [GitHub self-hosted runners](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners) on virtual machines.
+
+This charm simplifies the initial deployment and "day N" operations of GitHub self-hosted runners. The charm makes it easy to manage self-hosted runners with security and hardware resource usage in mind.
+
+Operating a self-hosted runner comes with [certain security concerns according to GitHub](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners#self-hosted-runner-security).
+Just like GitHub's, the self-hosted runners managed by the charm are isolated in a single-use virtual machine.
+
+The charm also upgrades dependencies on a schedule to mitigate security risks. This ensures the latest security patches are installed within minutes.
```

Comment:
Same comment as above about how this is done. Possibly here we just link to the place that explains how this is done.